data/remnant outfits.txt

```diff
@@ -341,6 +341,23 @@ outfit "Quantum Key Stone"
 
 
 
+# Scanner:
+
+outfit "Salvage Scanner"
+	category "Systems"
+	licenses
+		Remnant
+	cost 100000
+	thumbnail "outfit/salvage scanner"
+	"mass" 7
+	"outfit space" -7
+	"outfit scan power" 13
+	"outfit scan speed" 1
+	"tactical scan power" 84
```

Comment:
This seems to be a straight upgrade over the Tactical Scanner, with basically no downsides.

|Outfits|Tactical Scan|Outfit Scan|Size|S/M|Range/M|
|:---:|:---:|:---:|:---:|---:|---:|
|2x Tactical Scanner & 1x Outfit Scanner | 2*32 = 64 (800px)| 25 (500px) | 2*4 + 2 = 10 | 6.4, 2.5 | 80, 50|
|1x Salvage Scanner| 84 (916px) | 13 (360px) | 7 | 12.0, 1.8 | 130, 51|

Why would anyone with access to the Salvage, Tactical, and Outfit scanners not choose the Salvage Scanner?

Reply:
Perhaps increase OSP to 21 (i.e. 458 range) but reduce TSP to 49 (i.e. 700 range), to get per mass values of 3 and 7. (Human OS has 12.5, TS 8).

Reply:
Pretty much a straight upgrade, yes. If we had the capability to give alternate special abilities (like allowing the Remnant to target specific outfits with weapons or forcefully extract outfits without having to fully disable the ship, that'd be an interesting thing to consider.

That being said, I downgraded the outfit scanning down to 10. That should shift your values a bit.

Reply:
For comparison, the tactical scanner has 32/4=8 TSP/mass, the proposed salvage scanner 84/7=12, i.e. its 50% better even if it would have no outfit scan at all.

Usually when outfit A has only stat x, outfit B has only stat y, and outfit C has both x and y, then:
* A is the best option for people who only want x
* B is the best option for people who only want y
* C is the best option for people who want both x and y

(E.g. systems cores.)

Reply:
This is the case when adding and comparing a comparable outfit in the same tier, but doesn't really apply when one starts looking across tiers.

The Salvage scanner is a full upgrade to the Tier 1 human tactical scanner for a faction that puts a premium on knowledge and survival. And a key part of survival is knowing your enemy's strengths *before* committing to a fight. 

A better comparison would be to include the Scanning Module in this list. It boasts 25 power for outfit, tactical & cargo for a total of 75 in a 3 ton module. Double that, and one has 50 in each of tactical, outfit and cargo and still have 1 ton left over. That's 707 range for all three versus 914 and 360 for two. Overall, that's a lot more scanning potential than the Salvage Scanner can provide.

In terms of per mass calculations for the totals, that's 25 per ton for the scanning module versus 13.9 per ton for the Salvage Scanner. That's almost double the effective power. And that seems reasonable, given that the Heliarchs are more advanced than the Remnant in most areas. It also fits, as the Heliarchs haven't had an active war in a very long time, and so are mostly interested in law enforcement (which presumably includes scanning for illegal weapons and cargo). 

Remnant, on the other hand, have been in active warfare for over a century. At least initially, they would have been significantly underpowered compared to the Korath, and forced to rely on superior tactics and long-range information gathering to pick and choose targets. Even in the current point in time, they don't have a significant edge other than the fact that they can quickly bring in support to swarm hostiles (aka fast fleet spawns in their home territory). This (and missions where the player is asked to launch surveillance satellites) implies that they continue to rely on knowledge based tactics and strategies to give them an edge. 

A better question would be "If the players can reach the Remnant, and thus can almost certainly reach the Coalition, why would they use the Salvage Scanner when they could get the Scanner Module for far less space?" And the answer would be "Only if they just want more long range Tactical Scan capabilities." Since the Scanner Module is clearly far superior in as an overall sensor package.

Reply:
The difference is the new Heliarch Scanning Module is not for sale; the only way to obtain it is by plundering Heliarch ships (which are clearly stronger than the Remnant).

Reply:
Let's compare scan power / mass for all scanners:
* Asteroid Scanner: 40/6=6.667 (AS)
* Cargo Scanner: 9/1=9 (CS)
* Outfit Scanner: 25/2=12.5 (OS)
* Tactital Scanner: 32/4=8 (TS)
* *Navy Surveillance Pod*: 4/5=0.8 (CS), 16/5=3.2 (OS), 9/5=1.8 (TS)
* Remnant Salvage Scanner: 13/7=1.86 (OS), 84/7=12 (TS)
* *Heliarch Scanning Module*: 25/3=8.333 (CS, OS, TS)

Reply:
Thanks, that's helpful! 

And yep, the Salvage Scanner looks right about where I was aiming it in terms of both outfit scanning and tactical scanning.

(Just for the record, I expect the Wanderers to "rediscover" an antique tactical scanner at some point to go with their rediscovered warships, and the Korath should probably have one too. Although in both cases the power and attributes of those should reflect their respective cultures and priorities.)